.github/workflows/linkchecker.yaml

```diff
@@ -0,0 +1,26 @@
+name: Check links
```

Comment:
I have done this work also here https://github.com/canonical-web-and-design/cn.ubuntu.com/blob/master/.github/workflows/linkchecker.yaml (just sharing)